Tasks/PowerShellV2/powershell.ps1

```diff
@@ -48,7 +48,18 @@ try {
     # Generate the script contents.
     Write-Host (Get-VstsLocString -Key 'GeneratingScript')
     $contents = @()
+
+    # Explicitly set the default output stream preferences, rather then relying on PowerShell's defaults.
+    if ($env:system_debug -eq "true") {
+        $contents += "`$VerbosePreference = 'Continue'"
+    }
+    else {
+        $contents += "`$VerbosePreference = 'SilentlyContinue'"
+    }
+    $contents += "`$InformationPreference = 'Continue'"
+    $contents += "`$WarningPreference = 'Continue'"
```

Comment:
Nit: According to PowerShell documentation, the default value for `$WarningPreference` variable is [already set](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_preference_variables?view=powershell-7.1#warningpreference) to `Continue`. Could it make sense to remove it then?

Reply:
Yes, the default for `$WarningPreference` is `Continue`. I opted to explicitly set it here (rather than implicitly rely on PowerShell's defaults) for clarity and completeness.